### DISCUSSION TITLE
Helix editor on Fedora

### DISCUSSION THREAD

**Discussion by blinxen** [helix-editor/helix#8539] [General] [answered]
I am in the process of packaging the helix editor for Fedora and I should be done in the next couple of days (maybe a week or two). My question here before I release 23.05 is, will there be a new release soon? Or are you going to stick with 23.05 for a while?

I have not used `helix` until now but it looks very promising! Keep up the good work!!

(I hope this is the right communication canal to ask this kind of question)

**Comment by pascalkuthe** [comment, marked as answer]
Hi, for informal discussionr you are also welcome to join our matrix channel but GH discussions are just fine too.

If everything goes to plan we hope to cut a new release in the next couple days (or even this weekend) so it may make sense to wait until then. You could get started with the master branch. I don't think we will merge much (except for https://github.com/helix-editor/helix/pull/8396) so you could start packing the master branch for now and transition to the release with minimal effort as soon as its available.

A couple maintainers (including myself) use fedora so its great to see an official package!

**Reply by blinxen** [reply]
Just joined the matrix channel, did not know you had one :D

**Comment by archseer** [comment]
By the way there was another Fedora maintainer mentioning intent to package here: https://github.com/helix-editor/helix/discussions/8383

**Reply by blinxen** [reply]
Already spoke with him, he is not a Fedora maintainer yet but when he gets sponsored into one, he will help me with the maintenance.
